app/views/dashboards/_table_content.html.erb

```diff
@@ -1,4 +1,14 @@
-<!-- locals: title, table_type (for div id and tbody div id), pregnancies, sortable -->
+<!-- locals: title, table_type (for div id and tbody div id), pregnancies, sortable, autosortable -->
+
+<%
```

Comment:
Can you turn these into methods in `dashboards_helper`? Something like: 

```
def string_sort(local_assigns)
  local_assigns[:autosortable] ? ' data-sort="string"' : ''
end
```
etc